### ISSUE TITLE
Puzzled by the reproduction result

### ISSUE THREAD

**Issue by Choise-ieee** [EvgeneyBogatyrev/EfRLFN#2]
Dear author
I have sucessfully deplyed the coding in windows 10 system.
And this is my pip lists, which are according to the requires.txt:

Package                 Version
----------------------- -----------
absl-py                 2.4.0
aiohappyeyeballs        2.6.1
aiohttp                 3.13.3
aiosignal               1.4.0
albumentations          1.3.0
annotated-doc           0.0.4
anyio                   4.12.1
attrs                   25.4.0
brotlicffi              1.2.0.0
certifi                 2026.1.4
cffi                    2.0.0
charset-normalizer      3.4.4
click                   8.3.1
colorama                0.4.6
einops                  0.8.0
erqa                    1.1.2
ffms2                   0.4.5.5
filelock                3.20.3
frozenlist              1.8.0
fsspec                  2026.2.0
gmpy2                   2.2.2
grpcio                  1.66.1
h11                     0.16.0
h5py                    3.11.0
hf-xet                  1.2.0
httpcore                1.0.9
httpx                   0.28.1
huggingface_hub         1.4.1
idna                    3.11
imageio                 2.28.1
Jinja2                  3.1.6
joblib                  1.5.3
kornia                  0.7.0
lazy_loader             0.4
lightning-utilities     0.11.8
lit                     16.0.3
lpips                   0.1.4
Markdown                3.10.2
markdown-it-py          4.0.0
MarkupSafe              3.0.2
mdurl                   0.1.2
mkl_fft                 2.1.1
mkl_random              1.3.0
mkl-service             2.5.2
mpmath                  1.3.0
multidict               6.7.1
networkx                3.6.1
numpy                   1.24.3
opencv-python           4.8.0.74
opencv-python-headless  4.7.0.72
packaging               25.0
pandas                  2.0.1
Pillow                  9.5.0
pip                     26.0.1
propcache               0.4.1
protobuf                6.33.5
pycparser               2.23
Pygments                2.19.2
PySocks                 1.7.1
python-dateutil         2.9.0.post0
pytorch-lightning       1.9.0
pytz                    2025.2
PyWavelets              1.8.0
pywin32                 311
PyYAML                  6.0
qudida                  0.0.4
requests                2.32.5
rich                    14.3.2
safetensors             0.7.0
scikit-image            0.20.0
scikit-learn            1.2.2
scikit-video            1.1.11
scipy                   1.15.3
setuptools              80.10.2
shellingham             1.5.4
six                     1.17.0
sympy                   1.14.0
tensorboard             2.20.0
tensorboard-data-server 0.7.2
threadpoolctl           3.6.0
tifffile                2026.1.28
timm                    1.0.11
torch                   2.0.1
torch-tb-profiler       0.4.3
torchaudio              2.0.2
torchmetrics            1.5.1
torchvision             0.15.2
tqdm                    4.66.1
typer                   0.23.1
typer-slim              0.23.1
typing_extensions       4.15.0
tzdata                  2025.3
urllib3                 2.6.3
Werkzeug                3.1.5
wheel                   0.46.3
win_inet_pton           1.1.0
yarl                    1.22.0

But the inference results are bad as(>python inference_model.py -w weights/EfRLFN-2x-model.ph -s 2 -i images/input.png -o images/high_res.jpg )
And the input file is the input.png orinally, and the final picuture is the inference result.
<img width="480" height="270" alt="Image" src="https://github.com/user-attachments/assets/7e42a492-e569-4ee5-8337-e397ae5b533d" />
![Image](https://github.com/user-attachments/assets/be709ba2-e7c3-45dc-9cdb-002e40f59892)

And how can I achieve it ,Thanks for helping us, thanks

**Comment by Choise-ieee** [comment]
I have sucessfully inference:
python inference_model.py -w weigths/EfRLFN-4x-model.pt -s 4 -i images/input.png -o images/high_res.jpg
The readme has some errors inputings~~~~~

**Comment by Choise-ieee** [comment]
Thanks for your perfect work
I also sucessfuly deployed in C++ and onnx environment, and I also open-sourced in my github (https://github.com/Choise-ieee/EfRLFN-cpp-onnx), Thanks
